README.md

```diff
@@ -1,28 +1,25 @@
 # Introduction
 
-This repository contains all the annotation training data and models used by the PZH Natura 2000 remote sensing project.
-In short for this project we want to use machine learning on land satellite images to monitor each pixel distribution of various nature types in nature protected areas across time for various climate/nature policies.
-So the models here predicts for each pixel in a land satellite image a nature type, these types could include: Grass,Forest,Sand ..etc based on the learned annotations.
+This repository houses all the training data and models utilized in the PZH Natura 2000 remote sensing project. The aim of this project is to employ machine learning techniques on land satellite imagery to monitor the distribution of various nature types within protected areas over time, in alignment with diverse climate/nature policies. Therefore, the models herein predict the nature type for each pixel in a land satellite image, which could include Grass, Forest, Sand, etc., based on learned annotations.
 
-Annotated pixels pandas dataframe are stored in the data/annotations folder, as such it is open for the public to train their own model.
-This means that we took pixel values from in our case the SuperView land satellite images for annotated nature types.
-Based on these pixel value we want to predict what kind of nature type it is.
+The annotated pixels are stored as pandas dataframes in the data/annotations folder and are publicly available for anyone to train their own model. In essence, we used pixel values from the land satellite images to annotate various nature types. Our objective is to predict the type of nature based on these pixel values.
 
-Look in the data/annotation readme.md file for more information.
+For further information, please refer to the readme.md file in the data/annotation folder.
 
-3 types of pixel based models are here: a custom unsupervised supervised spectral contrast model, a keras deep learning network and random forest model trained on annotations.
+This repository includes three types of pixel-based models: a custom unsupervised spectral contrast model, a Keras deep learning network, and a random forest model trained on annotations. However, after a series of experiments, we found the random forest model trained on annotations to be the most effective and efficient for our project.
 
-After some experimentation for our project we went for the random forest model trained on annotations, giving us the best model results for the best execution performance.
+!Please note that we no longer support models other than the annotated supervised learning model.!
```

Comment:
I would not say "annotated supervised learning model", but "random forest model", to be consistent with the lines before and reduce confusion.